packages/website/ts/components/portal/portal.tsx

```diff
@@ -487,6 +528,51 @@ export class Portal extends React.Component<PortalProps, PortalState> {
         const isSmallScreen = this.props.screenWidth === ScreenWidths.Sm;
         return isSmallScreen;
     }
+
+    private _getCurrentTrackedTokens(): Token[] {
+        return this._getTrackedTokens(this.props.tokenByAddress);
+    }
+
+    private _getTrackedTokens(tokenByAddress: TokenByAddress): Token[] {
```

Comment:
it might be worth looking at some of these utility methods across our different components and identifying the ones that don't rely on the components state or props and moving those into a set of pure utility functions

Reply:
Yea... that or we need a Tokens class. I started moving some of this stuff in utils.ts.

Reply:
Just checked -- only used in Portal and TokenBalances.